packages/lit-dev-content/site/404.html

```diff
@@ -0,0 +1,18 @@
+---
+title: Not Found
+---
+
+{% extends 'default.html' %}
+
+{% block head %}
+  {% inlinecss "404.css" %}
+{% endblock %}
+
+{% block content %}
+
+<article id="container-404">
```

Comment:
TIL about [always including a hyphen in id](https://google.github.io/styleguide/htmlcssguide.html#:~:text=always%20include%20a%20hyphen)!